content/en/docs/collector/deployment/gateway.md

```diff
@@ -28,7 +28,7 @@ same collector instance where the tail sampling policy is applied.
 
 Let's have a look at such a case where we are using the load-balancing exporter:
 
-![Gateway deployment with load-balancing exporter](../../img/gateway-lb-sdk.svg)
+![Gateway deployment with load-balancing exporter](../../img/otel_gateway_lb_sdk_v2.svg)
```

Comment:
Per the following issue, please keep the kebab case in the filename:

- #780

Yeah, I know that the file names of all of the other files that you've updated the images for use snake case 🤷🏼‍♂️, but we're actually try to aim for kebab case as the standard.

(No need to change any of the other file names to kebab case in this PR, but if you do, that would be grand.)